### PR TITLE
Ide probe - remove `probe.resolvers.jbr.repositories` configs

### PR DESCRIPTION
**Pull request by LukaszKontowski**
Bump ide-probe to `0.43.0` so that we can remove `probe.resolvers.jbr.repositories` configs

### CODE REVIEW

**Comment by PawelLipski** [comment]
Pls target PRs to `develop`, not to `master`... `master` is meant for the released code

**Comment by PawelLipski** [comment]
(I've re-run the build after retargeting the PR)

File: .circleci/config.yml [hunk 1]
```diff
@@ -115,6 +127,15 @@ jobs:
           name: Verify binary compatibility with supported IntelliJ versions
           command: ./gradlew runPluginVerifier
 
+      - run:
```
Comment (PawelLipski) [inline .circleci/config.yml:130]: Whoops this needs to happen before compilation of tests

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # see https://docs.gradle.org/current/samples/sample_convention_plugins.html#things_to_note
 [versions]
 checker = "3.24.0"
-ideProbe = "0.41.0+28-68889008-SNAPSHOT"
+ideProbe = "0.0.0+530-68889008-SNAPSHOT"
```
Comment (PawelLipski) [inline gradle/libs.versions.toml:6]: Huh weird... looks that ide-probe is cloned without git tags, and/or your fork doesn't include the tags from the original VirtusLab/ide-probe repo?

Comment (PawelLipski) [reply gradle/libs.versions.toml:6]: Some combination of `git fetch --tags` and/or `git push --tags` should help

Comment (PawelLipski) [reply gradle/libs.versions.toml:6]: (albeit this is of minor importance anyway)

Comment (LukaszKontowski) [reply gradle/libs.versions.toml:6]: Ok, my fork has tags included - at leas it works for me locally. Won't fix it if no need for now

File: .circleci/config.yml [hunk 3]
```diff
@@ -148,7 +148,7 @@ jobs:
             # once both binary compatibility checks and UI tests for latest versions passed is close to zero.
             # Hence, to speed up the builds, we only run them for develop/master/hotfix/release and not on PRs.
             matches:
-              pattern: "^(develop|hotfix/.+|master|release/.+)$"
+              pattern: "^(develop|hotfix/.+|master|ide-probe|release/.+)$"
```
Comment (PawelLipski) [inline .circleci/config.yml:151]: This won't work as expected, https://virtuslab.github.io/tips/#regex/operators-precedence

Comment (PawelLipski) [reply .circleci/config.yml:151]: Plus also pls apply the same condition to ide-probe installation steps

Comment (LukaszKontowski) [reply .circleci/config.yml:151]: sure, thanks - fixing it

File: gradle/libs.versions.toml (hunk 2 above)
Comment (PawelLipski) [reply gradle/libs.versions.toml:6]: To be updated to 0.43.0 once released

Comment (LukaszKontowski) [reply gradle/libs.versions.toml:6]: If `0.43.0` will have https://github.com/VirtusLab/ide-probe/pull/257 included

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**